content/api/configuration.md

```diff
@@ -1,10 +1,1407 @@
 ---
 title: Configuration
 ---
-> configuration file
-> possible extensions, i. e. .babel.js
-> exporting a function and --env
-> returning a Promise
-> exporting multiple configurations
 
-> see also [[Using the Cli]]
+?> TODO: Add a reference to [[Using the Cli]]
+
+### Passing a Configuration
```

Comment:
Done (for external configurations as well). Moved these sections to the bottom for now as I think we should probably move them to their own pages.